test/testutils.jl

```diff
@@ -0,0 +1,150 @@
+# Note: Shapefile.jl saves Chains and Polygons as Multi
+# This function is used to work around this problem
+_isequal(d1::Domain, d2::Domain) = all(_isequal(g1, g2) for (g1, g2) in zip(d1, d2))
+_isequal(g1, g2) = g1 == g2
+_isequal(m1::Multi, m2::Multi) = m1 == m2
+_isequal(g, m::Multi) = _isequal(m, g)
+function _isequal(m::Multi, g)
+  gs = parent(m)
+  length(gs) == 1 && first(gs) == g
+end
+
+# Helper function to create a spatial reference
+spatialref(code) = AG.importUserInput(codestring(code))
+codestring(::Type{EPSG{Code}}) where {Code} = "EPSG:$Code"
+codestring(::Type{ESRI{Code}}) where {Code} = "ESRI:$Code"
+
+# Old GDAL implementation of projjsonstring for testing
+function gdalprojjsonstring(code; multiline=false)
+  spref = spatialref(code)
+  wktptr = Ref{Cstring}()
+  options = ["MULTILINE=$(multiline ? "YES" : "NO")"]
+  AG.GDAL.osrexporttoprojjson(spref, wktptr, options)
+  unsafe_string(wktptr[])
+end
+
+# Helper function to validate PROJJSON against schema
```

Comment:
This doesn't validate against the PROJJSON JSON schema, it just checks a few handpicked properties. Probably you want to use JSONSchema.jl to do this properly.

Reply:
Very useful package @nsajko , thanks for sharing!

@onyedikachi-david please try to use JSONSchema.jl in the tests.